victims/victims.go

```diff
@@ -56,7 +56,7 @@ type VictimAPICalls interface {
 
```

Comment:
Not really related with these changes but I noticed a small bug in line 158.
`glog.V(6).Infof("Killing %d running pods for %s %s", numPods, v.kind, v.name)`

I think it should be:
`glog.V(6).Infof("Killing %d running pods for %s %s", killNum, v.kind, v.name)`

Since you are already doing changes in the same class could you make this change too?

Reply:
Thanks for pointing this out. I will make this change too. Got a little distracted by work but I'll try to finish up this PR this week